### PR TITLE
Trash note

### PR DESCRIPTION
**Pull request by Sudhanshusoni45**
### Features added:
1. Trashed Notes
2. Delete Notes from trash
3. Trash context

#### login to test

### CODE REVIEW

**Comment by Jay-Gadhiya** [comment]
- Functionality working fine as mentioned.

**Comment by Jay-Gadhiya** [comment]
- You can add a title input box

File: src/context/trash-context.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const [trashState, setTrashState] = useState([]);
+
+  const moveToTrash = (trashedNote) => {
+    setTrashState((prevState) => [...prevState, trashedNote]);
+  };
+
+  const deleteFromTrash = (_id) => {
+    const newState = trashState.filter((item) => item._id !== _id);
+    setTrashState((prevState) => newState);
+  };
```
Comment (Jay-Gadhiya) [inline src/context/trash-context.js:18]: - suggestion: you can put this function outside of context.

**Review by Jay-Gadhiya — APPROVED** [review]

**Review by RahullRawat — APPROVED** [review]
You should try adding restore from trash functionality too if time persists.

Otherwise your code works fine. Great work @Sudhanshusoni45

**Pull request merged**